pokemon-league-participation-manager/src/test/java/cz/muni/fi/pa165/pokemon/league/participation/manager/dao/GymDAOTest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.muni.fi.pa165.pokemon.league.participation.manager.dao;
+
+import cz.muni.fi.pa165.pokemon.league.participation.manager.common.PersistenceApplicationContext;
+import cz.muni.fi.pa165.pokemon.league.participation.manager.entities.Gym;
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.PersistenceContext;
+import javax.transaction.Transactional;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import static org.mockito.Mockito.mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.test.context.ContextConfiguration;
+import org.testng.annotations.BeforeMethod;
+
+/**
+ * Unit test to gym data access object.
+ * 
+ * @author Tamás Rózsa 445653
+ */
+@ContextConfiguration(classes = PersistenceApplicationContext.class)
+@Transactional
+@RunWith(MockitoJUnitRunner.class)
+public class GymDAOTest {
```

Comment:
Remove this class. Not your implementation not committed form the author to master.